backend/server.py

```
@@ -0,0 +1,101 @@
+import asyncio
+
+from fastapi import FastAPI
+from contextlib import asynccontextmanager
+
+from src.utils.helpers import periodic_task
+from src.metrics.connection import get_questdb_pool
+from src.connectors.store import get_vectordb
+from src.utils.rag import *
```

Ruff (0.14.14)

[error] `from src.utils.rag import *` used; unable to detect undefined names

(F403)